src/transformers/models/whisper/generation_whisper.py

```diff
@@ -878,7 +986,7 @@ def _prepare_segments(prompt_ids, batch_size, generation_config):
 
         return current_segments
 
-    def _postprocess_outputs(self, seek_outputs, decoder_input_ids, return_token_timestamps, generation_config):
+    def _postprocess_outputs(self, seek_outputs, decoder_input_ids, return_token_timestamps, generation_config, is_shortform):
         # remove all previously passed decoder input ids
         if isinstance(seek_outputs, torch.Tensor):
```

Comment:
Not sure why this was marked as resolved, but not implemented in the latest commit:
https://github.com/huggingface/transformers/blob/38aa42d459ad5a904077ecf683fb5b670f4f2540/src/transformers/models/whisper/generation_whisper.py#L886

For all comments, you should either:
1. Implement the requested changes (if you agree with the proposition); or
2. Reply with a comment explaining why the changes should not be implemented or if an alternative design is required

=> only then should a comment be marked as resolved

Otherwise, when doing a second review, I'm just going to make the same comments as my first review, since I don't know why you haven't accepted them. This is a waste of time for both of us!